modules/microshift-restoring-data-backups.adoc

```diff
@@ -48,6 +48,8 @@ Replace `<my_manual_backup>` with the backup name that you used. Optional: You c
 $ sudo microshift restore /<mnt>/<other_backups_location>/<another_manual_backup>
 ----
 Replace `<other_backups_location>` with the directory you used and `<my_manual_backup>` with the backup name you used when creating the backup you are restoring.
-+
+
+. Restart the host. Restarting the host enables all workloads and pods to restart.
+
 .Verification
-* Verify that your backup is restored by restarting {microshift-short} and checking the data.
\ No newline at end of file
+* Use the `oc get pods -A` command to verify that the cluster is running, then check the restored data.
```

Comment:
Probably a small nit, but doesn't this command show that all of the pods are running in the cluster? It is still a valid check. @DanielFroehlich Can you please confirm ?

Reply:
@dhensel-rh 
right, this command shows all the pods in the cluster and their state. (Starting, Running, CrashLooping etc). it is the quickest and easiest way to check if everything is fine and up and running as expected. 

Hence: CONFIRMED! Itdoes make sense here to validate that the restore is okay. Its also okay NOT to give an example output, as this would highly depend on the workload / deployments of the customer.